netbox/extras/management/commands/dev.py

```diff
@@ -0,0 +1,22 @@
+import typing as t
+import threading
+from django.core.management import call_command
+from django.contrib.staticfiles.management.commands import runserver
+
+
+class Command(runserver.Command):
```

Comment:
Rather than introduce a command that's largely redundant to `runserver`, we can override `runserver` (as we do with `migrate` and `makemigrations`, under `utilities`) to add an optional command line argument that spawns the UI dev server. E.g. `manage.py runserver --with-dev-ui` or something like that.